lang/python/python-zope-interface/Makefile

```diff
@@ -29,7 +29,7 @@ define Package/python3-zope-interface
   SUBMENU:=Python
   TITLE:=Interfaces for Python
   URL:=https://github.com/zopefoundation/zope.interface
-  DEPENDS:=+python3-light
+  DEPENDS:=+python3-light +python3-logging
```

Comment:
Maybe we can say in the commit description, that there is a new dependency together with the test.sh. Otherwise, it looks like usual bump and there are many changes.

Reply:
Done!